.pre-commit-config.yaml

```diff
@@ -9,9 +9,10 @@ repos:
   - id: check-toml
   - id: check-yaml
   - id: check-json
-    exclude: .evergreen/atlas_data_lake/test.driverdata.json
+    exclude: '.evergreen/atlas_data_lake/test.driverdata.json'
```

Comment:
Let's revert changes to this file and `pyproject.toml` and make them part of the PR that adds `.git-ignore-revs`.

Reply:
The `pyproject.toml` changes are adding the ruff rules, wouldn't we want that in this PR?

Reply:
So that it doesn't get ignored as part of this commit.

Reply:
Ah right!